src/components/cc-addon-backups/cc-addon-backups.js

```diff
@@ -257,6 +257,7 @@ export class CcAddonBackups extends LitElement {
         ` : ''}
 
         ${emptyData ? html`
+          <div><span class=${classMap({ skeleton })}>${this._getDescription(providerId)}</span></div>
```

Comment:
**🎯 suggestion:** I think `skeleton` is always `false` when `emptyData` is true. :thinking: 

```suggestion
          <div>${this._getDescription(providerId)}</div>
```

Reply:
Yes indeed, thank you !